config.yaml

```diff
@@ -41,6 +41,25 @@ options:
       Enables the Hyper-V compute driver for clustering. This requires at least two Hyper-V compute
       joined to a cluster.
       NOTE: Available only in releases >= Mitaka
+  csv-performance-size:
+    type: string
+    default: ''
+    description: |
+      Represents the size occupied by the CSV requested from the s2d-proxy charm on the Performance
+      tier. In a Hyper-Converged scenario, each nova-hyperv unit requests a CSV to store the virtual machines.
+      Example: '20GB' or '20480MB'
```

Comment:
lets decide on the smallest unit of measure we want to support and just enforce that. MB or GB seem like good choices. Do you see any use for a volume less that 1 GB? Should we offer MB for greater granularity? what do you think?

I would hate to allow all known friently formats (MB, MiB, Mb, M, G, GB, GiB, etc...)